integrations/gateway-api-inference-extension/metadata.yaml

```diff
@@ -1,6 +1,5 @@
-sample_dashboards:
-  -
-    category: Inference Extension
-    id: gateway-api-inference-extension
-    display_name: Inference Extension Prometheus Overview
-    description: "This dashboard has charts displaying throughput, latency, cache usage, and other metrics for Inference Extension."
+id: gateway-api-inference-extension
+short_name: Inference Extension
+display_name: Inference Extension
+description: |
+  This dashboard has charts displaying throughput, latency, cache usage, and other metrics for Inference Extension.
```

Comment:
This should be a description of the **integration**, not of the **dashboard** (an integration can contain multiple dashboards). The integration description should be a description of what the application does and what metrics and logs it will collects, instead of talking about the charts.

![image](https://github.com/user-attachments/assets/c56d0768-cfff-47e3-8097-32859be51e04)